include/base/OpenMCCellAverageProblem.h

```diff
@@ -935,7 +946,7 @@ class OpenMCCellAverageProblem : public OpenMCProblemBase
   std::map<cellInfo, std::unordered_set<SubdomainID>> _cell_to_elem_subdomain;
 
   /// Mapping of elem subdomains to materials
-  std::map<SubdomainID, std::unordered_set<int32_t>> _subdomain_to_material;
+  std::map<SubdomainID, std::set<int32_t>> _subdomain_to_material;
```

Comment:
I think I did this so that when I later iterate through `_subdomain_to_material` that the print statements go through the subdomain IDs in order. But I'm not sure I actually tested it with `std::unordered_set` to see what the behavior was.

Reply:
Sounds good. I've got no preference myself in this case. Thanks!